src/org/omegat/core/machinetranslators/Google2Translate.java

```diff
@@ -155,22 +156,15 @@ private String unescapeHTML(String text) {
 
     @SuppressWarnings("unchecked")
     protected String getJsonResults(String json) {
-        Map<String, Object> rootNode;
+        JsonNode rootNode;
+        ObjectMapper mapper = new ObjectMapper();
         try {
-            rootNode = (Map<String, Object>) JsonParser.parse(json);
+            rootNode = mapper.readTree(json);
         } catch (Exception e) {
             Log.logErrorRB(e, "MT_JSON_ERROR");
             return OStrings.getString("MT_JSON_ERROR");
         }
-
-        try {
-            Map<String, Object> dataNode = (Map<String, Object>) rootNode.get("data");
-            List<Object> translationsList = (List<Object>) dataNode.get("translations");
-            Map<String, String> translationNode = (Map<String, String>) translationsList.get(0);
-            return translationNode.get("translatedText");
-        } catch (NullPointerException e) {
-            return null;
-        }
+        return rootNode.get("data").get("translations").get(0).get("translatedText").asText();
```

Comment:
This also has the same change in null handling as above.

Reply:
Because I found API document, I'd like to write response model and parse it explicitly.

Reply:
Done.